### DISCUSSION TITLE
Sobre a remoção do Terraform 0.11

### DISCUSSION THREAD

**Discussion by tigo17** [chgasparoto/curso-aws-com-terraform#33] [Q&A]
Oi professor, tudo bom?

O que motivou a remover o 0.11 do curso, acha que hoje não seria mais necessário, com o 0.14 daria para trabalhar com um terraform, 0.11, 12 e 13 etc?

Eu tinha gostado de como estava rs.. tinhas duas versões para ver, 11 e o mais novo, estava bem completo, eu podia até ter como referencia como era antes e como é hoje e derrepente poder usar algo do 11 no 14, por paticidade, mesmo recebendo um warning da vida etc.

Só queria sua opnião sinsera, só tendo o 14 lá hoje em dia já dá para trabalhar tranquilo?

Obs, eu nem tive um tempo para começar seu curso ainda, to em Cloud Amazon, Azure, Kubernetes etc. Terraform eu sei o básico, ai comprei seu curso para dar um deepdive.

Valeu.

**Comment by chgasparoto** [comment]
Fala @tigo17 , tudo certo e contigo?

A motivação é manter o curso atualizado, além de ter melhor performance, mais funcionalidades e mais bugs corrigidos.

Com a versão 0.14 você vai saber fazer tudo que a 0.11 oferece, a única e grande diferença é que a 0.11 você vai ter que usar muito mais interpolação para acessar os seus recursos, mas sinceramente não vejo vantagem nenhuma em utilizar essa versão tão antiga.

Abraço.
